packages/tunspace/tunspace.uc

```diff
@@ -228,6 +229,11 @@ function wg_replace_endpoint(ifname, cfg, next) {
   let srvcfg = cfg.wireguard_servers[next];
   let certopt = srvcfg.insecure_cert ? "--no-check-certificate" : "";
 
+  // bring interface down to prevent OLSR and Babel from spamming syslog.
+  if (0 != shell_command("ip link set down "+ifname)) {
```

Comment:
I'm really wondering whether we should write a netlink wrapper which abstracts this in a friendlier way than the native netlink library and contains quirks in case netlink is broken. But just a thought, LGTM

Reply:
Not sure I understand, the `Required key not available` message is straight from kmod-wireguard